src/commands/config/unset.ts

```diff
@@ -5,15 +5,15 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-import { Flags, loglevel } from '@salesforce/sf-plugins-core';
-import { Config, Messages, SfError } from '@salesforce/core';
-import { CONFIG_HELP_SECTION, ConfigCommand } from '../../config';
+import { Flags, loglevel, SfCommand, Ux } from '@salesforce/sf-plugins-core';
+import { Config, Messages } from '@salesforce/core';
+import { CONFIG_HELP_SECTION, buildFailureMsg, calculateSuggestion, output } from '../../config';
 import { SetConfigCommandResult } from './set';
 
 Messages.importMessagesDirectory(__dirname);
 const messages = Messages.loadMessages('@salesforce/plugin-settings', 'config.unset');
 
-export class UnSet extends ConfigCommand<SetConfigCommandResult> {
+export class UnSet extends SfCommand<SetConfigCommandResult> {
```

Comment:
The return type should either be specific to this command or if it will be reused, should be a name more suitable for reuse.